### PR TITLE
Add kourier and istio for 1.3 1.4

### PR DESCRIPTION
**Pull request by nak3**
As per title, this patch add kourier and istio for 1.3 1.4

### CODE REVIEW

**Comment by nak3** [comment]
/test pj-rehearse

infra

File: ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.3.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+- context_dir: .
+  dockerfile_path: Dockerfile.webhook
+  from: base
+  inputs:
+    openshift_release_golang-1.17:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.17
+  to: net-istio-webhook
+promotion:
+  name: knative-v1.3.0
```
Comment (skonto) [inline ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.3.yaml:33]: Should we update to latest eg. 1.3.2?

**Comment by skonto** [comment]
@nak3 we dont need the mirroring part?

**Pull request merged**